src/with-urql-client.tsx

```diff
@@ -110,27 +79,18 @@ function withUrqlClient<T = any, IP = any>(
         return pageProps;
       }
 
-      // Run the prepass step on AppTree. This will run all urql queries on the server.
-      await ssrPrepass(
-        <AppTree
-          pageProps={{
-            ...pageProps,
-            urqlClient,
-          }}
-        />,
-      );
+      const props = { ...pageProps, urqlClient };
+      const appTreeProps = isApp ? props : { pageProps: props };
```

Comment:
`_app` and Page components have different prop signatures, so we need to use the correct ones depending on what `AppTree` actually is.